dds_glossary/routes.py

```diff
@@ -0,0 +1,189 @@
+import logging
+from http import HTTPStatus
+
+from fastapi import APIRouter, Depends, FastAPI, Request
+from fastapi.responses import JSONResponse
+from starlette.templating import _TemplateResponse
+
+from . import __version__
+from .auth import get_api_key
+
+logger = logging.getLogger(__name__)
```

Comment:
@jsvgoncalves is this logger used?
Kindly make sure you setup the pre-commit hook locally.

Reply:
nope, but this is boilerplate I'd probably rather keep and avoid future usages of `logging.debug` etc.